src/data/graphql/typeDefs.ts

```diff
@@ -20,19 +20,22 @@ export default gql`
 
   type ParsedEvent {
     name: String!
-    topic: String!
     values: String!
     createdAt: Int!
-    from: String!
+    emmitedBy: String!
   }
 
-  type Transaction {
+  type ColonyAction {
     hash: String!
-    from: String!
-    to: String!
+    transactionInitiator: String!
```

Comment:
if we're refactoring everything from transaction to action, we probably should name it actionInititor

Reply:
Agree